src/Hocon.Immutable/Extensions/ValueGetterExtensions.cs

```diff
@@ -0,0 +1,340 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hocon.Immutable.Extensions
+{
+    public static class ValueGetterExtensions
```

Comment:
Contains static extension convenience helper functions to convert an immutable hocon literal into another primitive type. Sample use: `myHoconLiteral.GetDecimal()`.

Also contains backward compatible getter helper functions that follows the old `myHoconObject.GetInt("my.hocon.path")` API pattern.